web3/utils/abi.py

```diff
@@ -66,6 +66,11 @@ def get_abi_input_names(abi):
         return [arg['name'] for arg in abi['inputs']]
 
 
+def get_fallback_func_abi(contract_abi):
+    fallback_abis = filter_by_type('fallback', contract_abi)
+    return fallback_abis[0] if fallback_abis else None
```

Comment:
Instead of returning `None` here, I think it would be better to raise the `FallbackNotFound` exception.  Otherwise, any use of this function requires that we remember to *check* the return value which is easy to forget.  Forgetting to handle the exception will produce a much clearer error message than an un-expected `None` type floating around where we expected an ABI to be.

Reply:
Good point!